### PR TITLE
FLUID-5603: Add the API doc for the text to speech component

### PR DESCRIPTION
**Pull request by cindyli**
As well as updating docs for the preferences framework to reflect the recent upgrade of using relay components.

Note: Before merging in this pull request, need to consider if the infusion-docs repository should be tagged for infusion 1.5 docs.

### CODE REVIEW

File: src/documents/TextToSpeechAPI.md [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+
+# Text To Speech API #
+
+The **Text To Speech** component uses [Web Speech Synthesis API](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#tts-section) to queue up and read texts.
+
+## Browser Support ##
+The Text To Speech component can be used in browsers that support [Web Speech Synthesis API](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#tts-section) is supported. At the moment when this document is written, Mar 4 2015, these browsers include:
+* Chrome 33+ 
+* Chrome for Android 40+
+* Safari 7.1+
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: Isn't Safari on iOS also supported?

File: src/documents/TextToSpeechAPI.md [hunk 2]
```diff
@@ -0,0 +1,580 @@
...
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.queueSpeech(text, interrupt, options);</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>queueSpeech</code> method allows you to append the text to the end of the queue to begin speaking after the other texts in the queue have been spoken. This method can also be used to remove all texts from the queue by passing in the `interrupt` argument with a true value.
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: This last sentence might be better worded as "Setting the 'interrupt' argument to true will remove all texts from the queue before adding 'queueSpeech'.

File: src/documents/TextToSpeechAPI.md [hunk 3]
```diff
@@ -0,0 +1,580 @@
...
+            <th>Parameters</th>
+            <td>
+                <dl>
+                    <dt>text</dt>
+                    <dd>
+                        The string of the text to be appended to the end of the queue to be spoken.
+                    </dd>
+                    <dt>interrupt</dt>
+                    <dd>
+                        An optional boolean value. Passing in a true value to remove all texts from the queue. Passing in the false value doesn't interfere the component state.
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: We should say what the default is.

File: src/documents/TextToSpeechAPI.md [hunk 4]
```diff
@@ -0,0 +1,580 @@
...
+                    </dd>
+                    <dt>options</dt>
+                    <dd>
+                        An optional javascript object. To config the internal `SpeechSynthesisUtterance` instance. Refer to <a href="#utteranceopts-options">utteranceOpts Options</a> for the options that can be defined in this object.
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: It's not clear what the relationship is between utteranceOpts and this method. Are you saying that anything that can go in utteranceOpts can also be used as an option to this method?

File: src/documents/TextToSpeechAPI.md [hunk 5]
```diff
@@ -0,0 +1,580 @@
...
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.pause();</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>pause</code> method allows you to immediately pause any texts that are being spoken..
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: Extra period at end of sentence here.

File: src/documents/TextToSpeechAPI.md [hunk 6]
```diff
@@ -0,0 +1,580 @@
...
+The events fired by the Text To Speech component are described below.
+
+### `onStart` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                This event fires when the text has begun to be spoken.
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: Do these start and stop events fire for each individual text in the queue? Or would there only be one stop event when the queue is finally emptied?

File: src/documents/TextToSpeechAPI.md [hunk 7]
```diff
@@ -0,0 +1,580 @@
...
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## `utteranceOpts` Options ##
+
+_**Note:** If needed, please read the [Component Configuration Options](ComponentConfigurationOptions.md) document for a full description of infusion component options._
+
+The only option supported by the Text To Speech component is `utteranceOpts`This option is a javascript object that contains a number of attributes that users can use to define the behaviour of their `SpeechSynthesisUtterance` instances. [The `SpeechSynthesisUtterance` instance](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#utterance-attributes) is a part of the web speech API that the Text To Speech component interacts with. These attributes include:
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: The first sentence in this paragraph is missing a period at the end.

Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: I might reword this as

The only option supported by the Text To Speech component is `utteranceOpts`This option is a javascript object that contains attributes that users can use to define the behaviour of the [`SpeechSynthesisUtterance` instance](link here) (a part of the web speech API that the Text To Speech component interacts with). These attributes include:

How does that look?

File: src/documents/TextToSpeechAPI.md [hunk 8]
```diff
@@ -0,0 +1,580 @@
...
+</table>
+
+### `lang` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>lang</code> option gives you the ability to specify the language of the text.
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: We should state what form is expected - language code? based on what specification? ISO?

File: src/documents/TextToSpeechAPI.md [hunk 9]
```diff
@@ -0,0 +1,580 @@
...
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.getVoices();</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>getVoices</code> method allows you to get a list of all the voices that are supported by the browser.
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: What is the form of this list? An array of strings? what's in the strings?

File: src/documents/TextToSpeechAPI.md [hunk 10]
```diff
@@ -0,0 +1,580 @@
...
+</table>
+
+### `voiceURI` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>voiceURI</code> option specifies speech synthesis voice and the location of the speech synthesis service that the web application wishes to use.
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: It looks like this is actually an object with several properties. We should probably explain the properties.

File: src/documents/TextToSpeechAPI.md [hunk 11]
```diff
@@ -0,0 +1,580 @@
...
+</table>
+
+### `rate` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>rate</code> option allows you to define the speed at which the text should be spoken. This is a float value between 0 and 10.
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: If this is a float value, maybe the example should reinforce that fact by actually using a float value.

File: src/documents/TextToSpeechAPI.md [hunk 12]
```diff
@@ -0,0 +1,580 @@
...
+</table>
+
+### `pitch` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>pitch</code> option allows you to control how high or low the text is spoken. This is a float value between 0 and 2.
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: Again, if this is a float, the example should use a float (the Default, too).

File: src/documents/Enactors.md [hunk 13]
```diff
@@ -23,18 +23,18 @@ Each of these is explained below.
 
 ### Grade ###
 
-Enactors must be defined using the `fluid.prefs.enactors` [grade](ComponentGrades.md), as shown in the following code block:
+Enactors must be defined using the `fluid.prefs.enactor` [grade](ComponentGrades.md), as shown in the following code block:
```
Comment (acheetham) [inline src/documents/Enactors.md]: @cindyli, could you start a document listing API changes from 1.5 to ... well, I guess it would be 2.0, wouldn't it? Things like this enactor name change, the xxRelayComponent name changes, and anything else you know of. It doesn't have to be super final, just a place where we start recording this so we don't forget, and so we don't have a huge task trying to remember everything when release time comes

File: src/documents/Enactors.md [hunk 14]
```diff
@@ -23,18 +23,18 @@ Each of these is explained below.
...
+Enactors are, by default, [model components](tutorial-gettingStartedWithInfusion/ModelComponents.md) and [evented components](tutorial-gettingStartedWithInfusion/EventedComponents.md), so they automatically provides support for a model and for events. If other support is needed, other grades can be added. For example, if the enactor will be operating on the DOM, the [`fluid.viewRelayComponent`](https://github.com/fluid-project/infusion/blob/infusion-1.5/src/framework/core/js/FluidView.js#L40-L42) grade should be used, and the `selectors` option should be provided, as shown in the following example:
```
Comment (acheetham) [inline src/documents/Enactors.md]: What should we do about this link that's hard-coded to 1.5? Should we change it to master for now?

File: src/documents/IoCReferences.md [hunk 15]
```diff
@@ -193,7 +193,6 @@ fluid.defaults("fluid.videoPlayer.controllers", {
             container: "{controllers}.dom.scrubberContainer",
             options: {
                 model: "{controllers}.model",
-                applier: "{controllers}.applier",
```
Comment (acheetham) [inline src/documents/IoCReferences.md]: Why don't we specify the applier anymore? This might be a change that should be explained in the "API changes from 1.5 to 2.0" document.

File: src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md [hunk 16]
```diff
@@ -123,7 +123,7 @@ fluid.defaults("gpii.pmt.enactors.magnification", {
     ....
     modelListeners: {
         "magnificationFactor": {
-            funcName: "gpii.pmt.enactors.magnification.magnify",
+            listener: "gpii.pmt.enactors.magnification.magnify",
```
Comment (acheetham) [inline src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md]: Is this an API change, or was it a mistake in the documentation?

File: src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md [hunk 17]
```diff
@@ -227,19 +224,23 @@ fluid.defaults("fluid.prefs.enactors.tableOfContents", {
         afterTocRender: null,
         onLateRefreshRelay: null
     },
-    listeners: {
-        onCreate: {
+    modelListeners: {
+        toc: {
             listener: "{that}.applyToc",
-            args: "{that}.model.value"
+            args: ["{change}.value"]
```
Comment (acheetham) [inline src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md]: Should this be value or toc?

File: src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md (hunk 16 above)
Comment (cindyli) [inline src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md]: According to the conversation with Bosmon (https://botbot.me/freenode/fluid-work/2015-03-10/?tz=America/Havana, 1:30PM), both "funcName" and "listener" are currently valid and equivalent. Since the plan is to gradually eliminate "listener", I will replace "listener" here back to "funcName", as well as other places in docs.

File: src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md (hunk 17 above)
Comment (cindyli) [inline src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md]: Should be toc. At the line 187, the prefs map links the default primary schema value with "model.toc".

**Comment by cindyli** [comment]
@acheetham, this pull request is ready for another look. Thanks.

File: src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md (hunk 17 above)
Comment (acheetham) [inline src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md]: Still confused. If it should be toc, why is it value?

File: src/documents/TextToSpeechAPI.md [hunk 18]
```diff
@@ -0,0 +1,583 @@
...
+</table>
+
+### `voiceURI` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>voiceURI</code> option is a A string that specifies the speech synthesis voice and the location of the speech synthesis service that the web application wishes to use. Calling the <a href=""#getvoices"">getVoices</a> method returns an array of all available voices, from which you can get the value of the <code>voiceURI</code> attribute.<br />
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: You have an extra "A" near the beginning of this sentence.

File: src/documents/TextToSpeechAPI.md [hunk 19]
```diff
@@ -0,0 +1,583 @@
...
+                <strong>Note</strong>: In Chrome and Safari, the <code>voiceURI</code> attribute is named <code>voice</code> instead.
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: This is confusing, since the speech API is currently only support in Chrome and Safari. Should the reader use voice, or voiceURI?

File: src/documents/TextToSpeechAPI.md [hunk 20]
```diff
@@ -0,0 +1,583 @@
...
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.queueSpeech(text, interrupt, options);</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>queueSpeech</code> method allows you to append the text to the end of the queue to begin speaking after the other texts in the queue have been spoken. Setting the `interrupt` argument to true will remove all texts from the queue before adding the text to the queue.
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: This is still a bit unclear because of the repetition of 'text' (I realize now that my original suggestion was wrong). How about changing "adding the text to the queue" to "adding 'text' to the queue" ? Hopefully the quotes will make it more clear that this use of 'text' is a reference to the argument.

File: src/documents/TextToSpeechAPI.md (hunk 19 above)
Comment (cindyli) [inline src/documents/TextToSpeechAPI.md]: voiceURI is the official attribute specified in the W3C web speech API specification (https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#dfn-voicelang), which is supposed to be followed by browsers. I figure it's relatively safer to use what's in the spec and point out the exception with Chrome and Safari.

File: src/documents/TextToSpeechAPI.md [hunk 21]
```diff
@@ -0,0 +1,583 @@
...
+title: Text To Speech API
+layout: default
+---
+
+# Text To Speech API #
+
+The **Text To Speech** component uses [Web Speech Synthesis API](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#tts-section) to queue up and read texts.
+
+## Browser Support ##
+The Text To Speech component can be used in browsers that support [Web Speech Synthesis API](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#tts-section) is supported. At the moment when this document is written, Mar 4 2015, these browsers include:
```
Comment (jobara) [inline src/documents/TextToSpeechAPI.md]: The "is supported" at the end of the first sentence is unnecessary as you've already mentioned "that support" earlier.

For the second sentence you may want to phrase it as "At the time of writing, Mar 4 2015, these browsers include:" and maybe include a link to [caniuse](http://caniuse.com/#feat=web-speech)

File: src/documents/TextToSpeechAPI.md (hunk 20 above)
Comment (jobara) [inline src/documents/TextToSpeechAPI.md]: because this is in a table, the code needs to be in `<code>` tags.

File: src/documents/TextToSpeechAPI.md [hunk 22]
```diff
@@ -0,0 +1,583 @@
...
+                The <code>queueSpeech</code> method allows you to append the text to the end of the queue to begin speaking after the other texts in the queue have been spoken. Setting the `interrupt` argument to true will remove all texts from the queue before adding the text to the queue.
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dl>
+                    <dt>text</dt>
+                    <dd>
+                        The string of the text to be appended to the end of the queue to be spoken.
```
Comment (jobara) [inline src/documents/TextToSpeechAPI.md]: Maybe rephrase this as.

"The string of text to be appended to the queue and spoken."

File: src/documents/TextToSpeechAPI.md [hunk 23]
```diff
@@ -0,0 +1,583 @@
...
+                    </dd>
+                    <dt>interrupt</dt>
+                    <dd>
+                        An optional boolean value. The default value is false. Setting it to true will remove all texts from the queue before adding the text. Setting it to false does not interfere the component state.
```
Comment (jobara) [inline src/documents/TextToSpeechAPI.md]: For the last sentence, maybe rephrase as.

"Setting it to false will not affect previously queued text."

File: src/documents/TextToSpeechAPI.md [hunk 24]
```diff
@@ -0,0 +1,583 @@
...
+                    </dd>
+                    <dt>options</dt>
+                    <dd>
+                        An optional javascript object. To config the internal `SpeechSynthesisUtterance` instance. Anything that can go in <a href="#utteranceopts-options">utteranceOpts</a> can also be used as an option to this method.
```
Comment (jobara) [inline src/documents/TextToSpeechAPI.md]: "SpeechSynthesisUtterance" should be in `<code>` block. 

Also we may want to rephrase this like:

"An optional javascript object. Allows for the configuration of the specific <code>SpeechSynthesisUtterance</code> instance used for this particular text. The configuration passed in here takes the same form as <a href="#utteranceopts-options">utteranceOpts</a> and will override them for this instance only."

File: src/documents/TextToSpeechAPI.md [hunk 25]
```diff
@@ -0,0 +1,583 @@
...
+The only option supported by the Text To Speech component is `utteranceOpts`. This option is a javascript object that contains attributes that users can use to define the behaviour of the [SpeechSynthesisUtterance instance](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#utterance-attributes) (a part of the web speech API that the Text To Speech component interacts with). These attributes include:
+
+### `text` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>text</code> option allows you to set the text that you wish to be spoken. Be careful with this option as it will override any text that was previously passed.
```
Comment (jobara) [inline src/documents/TextToSpeechAPI.md]: Maybe the "Be careful..." part should be written as a note.

File: src/documents/TextToSpeechAPI.md [hunk 26]
```diff
@@ -0,0 +1,583 @@
...
+        </tr>
+    </tbody>
+</table>
+
+## `utteranceOpts` Options ##
+
+_**Note:** If needed, please read the [Component Configuration Options](ComponentConfigurationOptions.md) document for a full description of infusion component options._
+
+The only option supported by the Text To Speech component is `utteranceOpts`. This option is a javascript object that contains attributes that users can use to define the behaviour of the [SpeechSynthesisUtterance instance](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#utterance-attributes) (a part of the web speech API that the Text To Speech component interacts with). These attributes include:
+
```
Comment (jobara) [inline src/documents/TextToSpeechAPI.md]: A note should also be added that not all speech synthesizers support all of the options, and some may take different ranges.

**Comment by cindyli** [comment]
Thanks for comments. All addressed and ready for another round of review.

File: src/documents/Enactors.md [hunk 27]
```diff
@@ -23,18 +23,18 @@ Each of these is explained below.
...
+Enactors are, by default, [model components](tutorial-gettingStartedWithInfusion/ModelComponents.md) and [evented components](tutorial-gettingStartedWithInfusion/EventedComponents.md), so they automatically provides support for a model and for events. If other support is needed, other grades can be added. For example, if the enactor will be operating on the DOM, the [`fluid.viewRelayComponent`](https://github.com/fluid-project/infusion/blob/master/src/framework/core/js/FluidView.js#L40-L42) grade should be used, and the `selectors` option should be provided, as shown in the following example:
```
Comment (jobara) [inline src/documents/Enactors.md]: Based on a quick look at the code, it seems that they are `fluid.viewRelayComponent` or based on the base grade `fluid.prefs.enactor` they are `fluid.standardRelayComponent`. Also it should be "... so they automatically provide..." not "provides".

File: src/documents/TextToSpeechAPI.md [hunk 28]
```diff
@@ -0,0 +1,586 @@
...
+# Text To Speech API #
+
+The **Text To Speech** component uses [Web Speech Synthesis API](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#tts-section) to queue up and read texts.
+
+## Browser Support ##
+The Text To Speech component can be used in browsers that support [Web Speech Synthesis API](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#tts-section). At the time of writing, Mar 4 2015, these browsers include:
+* Chrome 31+ 
+* Chrome for Android 40+
+* Safari 7.1+
+* iOS Safari 7.1+
```
Comment (jobara) [inline src/documents/TextToSpeechAPI.md]: did you want to include a link to canisuse? http://caniuse.com/#feat=web-speech

File: src/documents/TextToSpeechAPI.md [hunk 29]
```diff
@@ -0,0 +1,586 @@
...
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.queueSpeech(text, interrupt, options);</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>queueSpeech</code> method allows you to append `text` to the end of the queue to begin speaking after the other texts in the queue have been spoken. Setting the <code>interrupt</code> argument to true will remove all texts from the queue before adding <code>text</code> to the queue.
```
Comment (jobara) [inline src/documents/TextToSpeechAPI.md]: There's another instance of the word "text" using "`" instead of `<code>`

File: src/documents/TextToSpeechAPI.md [hunk 30]
```diff
@@ -0,0 +1,586 @@
...
+</table>
+
+## `utteranceOpts` Options ##
+
+_**Note:** If needed, please read the [Component Configuration Options](ComponentConfigurationOptions.md) document for a full description of infusion component options._
+
+The only option supported by the Text To Speech component is `utteranceOpts`. This option is a javascript object that contains attributes that users can use to define the behaviour of the [SpeechSynthesisUtterance instance](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#utterance-attributes) (a part of the web speech API that the Text To Speech component interacts with). These attributes include:
+
+_**Note:** Not all browsers support all these options and some browsers may take different ranges._
+
```
Comment (jobara) [inline src/documents/TextToSpeechAPI.md]: It's not really a browser issue, but a speech synthesizer issue. A browser may support many different speech synthesizers.

File: src/documents/TextToSpeechAPI.md [hunk 31]
```diff
@@ -0,0 +1,586 @@
...
+</table>
+
+### `voiceURI` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>voiceURI</code> option is a string that specifies the speech synthesis voice and the location of the speech synthesis service that the web application wishes to use. Calling the <a href=""#getvoices"">getVoices</a> method returns an array of all available voices, from which you can get the value of the <code>voiceURI</code> attribute.<br />
```
Comment (jobara) [inline src/documents/TextToSpeechAPI.md]: There's an extra set of quotes around the url to getVoices which is causing the link to break.

**Comment by cindyli** [comment]
@acheetham and @jobara, this pull request is ready again.

File: src/documents/TextToSpeechAPI.md [hunk 32]
```diff
@@ -0,0 +1,590 @@
...
+    }
+});
+</code>
+</pre>
+            </td>
+        </tr>
+        <tr>
+            <th>See also</th>
+            <td>
+                <a href="#utteranceopts-options">Utteranceopts Options</a>
```
Comment (acheetham) [inline src/documents/TextToSpeechAPI.md]: This link isn't working.

File: src/documents/tutorial-migratingToInfusion2.0/UIOptionsMigration.md [hunk 33]
```diff
@@ -0,0 +1,133 @@
+---
+title: UI Options 2.0 Migration
```
Comment (acheetham) [inline src/documents/tutorial-migratingToInfusion2.0/UIOptionsMigration.md]: This information is general 1.5 to 2.0 changes, not a tutorial specific to UIOptions. It should probably be in a top-level file called "APIChangesFrom1_5To2_0.md"

**Comment by jobara** [comment]
Replaced by https://github.com/fluid-project/infusion-docs/pull/43
